docs/user-guide/work-packages/README.md

```diff
@@ -30,12 +30,13 @@ Work packages can be displayed in a projects timeline, e.g. as a milestone or a
 | Topic                                                        | Content                                                      |
 | ------------------------------------------------------------ | :----------------------------------------------------------- |
 | [Work packages views](work-package-views)                    | What is the difference between the work packages views: list view, split screen view, details view? |
-| [Create a work package](create-work-package)                 | How to create a new work package in OpenProject?             |
-| [Edit work package](edit-work-package)                       | How to edit a work package in OpenProject?                   |
-| [Copy, move, delete](copy-move-delete)                       | How to copy, move, delete a work package?                    |
-| [Work package table configuration](work-package-table-configuration) | How to configure the work package table (columns, filters, group by, etc.)? |
-| [Exporting](exporting)                                       | How to export work packages for other tools such as Microsoft Excel? |
-| [Work package relations and hierarchies](work-package-relations-hierarchies) | How to create work package relations and hierarchies?        |
+| [Create a work package](create-work-package)                 | How to create a new work package in OpenProject              |
+| [Set and change dates](set-and-change-dates)                 | How to set and change start and finish dates of work packages in OpenProject |
```

Comment:
The path fragment is different, the link should be 
`[Set and change dates](set-change-dates)`

Reply:
Fixed, thank you.